ci/deploy_app.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+IMAGE_TAG="${IMAGE_TAG:=latest}"
+REGION="${REGION:=europe-west2}"
+MIN_INSTANCES="${MIN_INSTANCES:=1}"
+MAX_INSTANCES="${MAX_INSTANCES:=1}"
+
+gcloud beta run deploy eq-questionnaire-launcher \
+    --project="${PROJECT_ID}" --region="${REGION}" --concurrency=250 --min-instances="${MIN_INSTANCES}" --max-instances="${MAX_INSTANCES}" \
+    --port=8000 --cpu=1 --memory=256M \
+    --image="${DOCKER_REGISTRY}/eq-questionnaire-launcher:${IMAGE_TAG}" --platform=managed --allow-unauthenticated \
+    --set-env-vars SURVEY_RUNNER_URL="https://${RUNNER_URL}"
```

Comment:
The full runner URL should be passed, rather than get concatenated with `https://` in this script.

Reply:
Ok, made relevant changes in pipeline so no concatenation here now.